examples/tutorials/05_creating_CNN.ipynb

```diff
@@ -13,8 +13,7 @@
    "source": [
```

Comment:
<p>There are no random_rotate and random_scale</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/476/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>